packages/react-next/etc/react-next.api.md

```diff
@@ -1285,8 +1282,6 @@ export interface ISpinButtonProps extends React.HTMLAttributes<HTMLDivElement> {
     defaultValue?: string;
     disabled?: boolean;
     downArrowButtonStyles?: Partial<IButtonStyles>;
-    // Warning: (ae-forgotten-export) The symbol "ISpinButtonClassNames" needs to be exported by the entry point index.d.ts
-    getClassNames?: (theme: ITheme, disabled: boolean, isFocused: boolean, keyboardSpinDirection: KeyboardSpinDirection, labelPosition?: Position, className?: string) => ISpinButtonClassNames;

```

Comment:
please update notes [here](https://github.com/microsoft/fluentui/blob/master/packages/react-next/release.md) to include this breaking change

Reply:
@czearing - could you get this resolved quickly so we can merge this PR :)